clients/cloud/python/README.md

```diff
@@ -0,0 +1,55 @@
+# Overview
+
+Produce messages to and consume messages from Confluent Cloud using Confluent Python Client for Apache Kafka
+
+
+# Prerequisites
+
+* Install Confluent's Python Client for Apache Kafka: https://github.com/confluentinc/confluent-kafka-python
+* File with client configuration parameters to connect to your Confluent Cloud instance
+
+
+# Usage
+
+$ cat ~/.ccloud/librdkafka.config
+bootstrap.servers=<>
+sasl.username=<>
+sasl.password=<>
+
+
+$ ./producer.py ~/.ccloud/librdkafka.config test1                      
```

Comment:
I got the following output:
```

%3|1544080882.161|FAIL|rdkafka#producer-1| [thrd:sasl_ssl://b1-pkc-epvye.us-central1.gcp.confluent.cloud:9092/1]: sasl_ssl://b1-pkc-epvye.us-central1.gcp.confluent.cloud:9092/1: SASL authentication failure: Disconnected: check client PLAIN credentials and broker logs
%3|1544080882.161|ERROR|rdkafka#producer-1| [thrd:sasl_ssl://b1-pkc-epvye.us-central1.gcp.confluent.cloud:9092/1]: sasl_ssl://b1-pkc-epvye.us-central1.gcp.confluent.cloud:9092/1: SASL authentication failure: Disconnected: check client PLAIN credentials and broker logs
%3|1544080882.164|FAIL|rdkafka#producer-2| [thrd:sasl_ssl://b1-pkc-epvye.us-central1.gcp.confluent.cloud:9092/1]: sasl_ssl://b1-pkc-epvye.us-central1.gcp.confluent.cloud:9092/1: SASL authentication failure: Disconnected: check client PLAIN credentials and broker logs
%3|1544080882.164|ERROR|rdkafka#producer-2| [thrd:sasl_ssl://b1-pkc-epvye.us-central1.gcp.confluent.cloud:9092/1]: sasl_ssl://b1-pkc-epvye.us-central1.gcp.confluent.cloud:9092/1: SASL authentication failure: Disconnected: check client PLAIN credentials and broker logs
Topic test1 created
alice 	 {"count": 0}
alice 	 {"count": 1}
alice 	 {"count": 2}
alice 	 {"count": 3}
alice 	 {"count": 4}
alice 	 {"count": 5}
alice 	 {"count": 6}
alice 	 {"count": 7}
alice 	 {"count": 8}
alice 	 {"count": 9}
%3|1544080884.647|FAIL|rdkafka#producer-1| [thrd:sasl_ssl://b1-pkc-epvye.us-central1.gcp.confluent.cloud:9092/1]: sasl_ssl://b1-pkc-epvye.us-central1.gcp.confluent.cloud:9092/1: SASL authentication failure: Disconnected: check client PLAIN credentials and broker logs
%3|1544080884.647|ERROR|rdkafka#producer-1| [thrd:sasl_ssl://b1-pkc-epvye.us-central1.gcp.confluent.cloud:9092/1]: sasl_ssl://b1-pkc-epvye.us-central1.gcp.confluent.cloud:9092/1: SASL authentication failure: Disconnected: check client PLAIN credentials and broker logs
```

Reply:
I'm with @rnpridgeon: Where not overly complex, the example should show what good client code looks like. This one-line is something I'd want to know about (how the logger is passed to the producer).

If we are fair, the cloud part is only: Copy-paste stuff from the web and read configuration from a file. The rest is "how to use Confluent-python client", and getting the logs is an important part IMO.

Reply:
As discussed internally, we are omitting logging